spec/lib/monitorable_resource_spec.rb

```diff
@@ -53,7 +53,7 @@
       expect(session[:ssh_session]).to receive(:close)
       expect(Thread).to receive(:new).and_call_original
       expect(session[:ssh_session]).to receive(:loop).and_raise(StandardError)
-      expect(Clog).to receive(:emit).twice.and_call_original
+      expect(Clog).to receive(:emit).exactly(3).and_call_original
```

Comment:
If you really want to test it's three times, there's a method for that.

```suggestion
      expect(Clog).to receive(:emit).thrice.and_call_original
```

However, if you don't care about exactly how many times:

```suggestion
      expect(Clog).to receive(:emit).at_least(:once).and_call_original
```